test/powershell/Modules/Microsoft.PowerShell.Utility/Select-String.Tests.ps1

```diff
@@ -71,6 +71,33 @@ Describe "Select-String" -Tags "CI" {
 	    $testinputone | Select-String -Pattern "goodbye" -NotMatch | Should -Be "hello", "hello"
 	}
 
+	it "Should return an array of matching strings with the first match highlighted when Emphasize is used" {
+		if ($IsWindows)
```

Comment:
Doesn't  Windows support VT?

Reply:
Yeah rather than doing `$IsWindows` you can do:

```
$Host.UI.SupportsVirtualTerminal
```

Which returns a bool.

Reply:
The first version of Windows 10 (1507) did not support VT. Probably not many folks still using that version but $Host.UI.SupportsVirtualTerminal just as well.  And you may be running in a different host than conhost.

Reply:
But those hosts would just not display the color, right?

Would also be best to consider https://no-color.org/